cmd/project/samples.go

```diff
@@ -66,6 +71,12 @@ func NewSamplesCommand(clients *shared.ClientFactory) *cobra.Command {
 // runSamplesCommand prompts for a sample then clones with the create command
 func runSamplesCommand(clients *shared.ClientFactory, cmd *cobra.Command, args []string) error {
 	ctx := cmd.Context()
+
+	// DEPRECATED(semver:major): Prefer the create command when repository details are known
+	if cmd.Flag("branch").Changed || cmd.Flag("template").Changed {
+		clients.IO.PrintWarning(ctx, "DEPRECATED: The '--branch' and '--template' flags are deprecated for the 'samples' command; use the 'create' command instead")
```

Comment:
suggestion: Based on https://github.com/slackapi/slack-cli/pull/194, do you think we should start to use backticks instead of single quotes for the flags and commands?
```suggestion
		clients.IO.PrintWarning(ctx, "DEPRECATED: The `--branch` and `--template` flags are deprecated for the `samples` command; use the `create` command instead")
```